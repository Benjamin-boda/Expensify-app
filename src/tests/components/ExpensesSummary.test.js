import { shallow } from "enzyme";
import React from "react";
import { ExpensesSummary } from "../../components/ExpensesSummary";

test("should correctly render Expenses summary with 1 expense ", () => {
    const wrapper = shallow(<ExpensesSummary expenseCount={1} expensesTotal={123} />)
    expect(wrapper).toMatchSnapshot();
});

test("should correctly render Expenses summary with multiple expense ", () => {
    const wrapper = shallow(<ExpensesSummary expenseCount={14} expensesTotal={1239899} />)
    expect(wrapper).toMatchSnapshot();
});