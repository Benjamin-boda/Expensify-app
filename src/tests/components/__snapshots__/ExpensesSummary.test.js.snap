// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should correctly render Expenses summary with 1 expense  1`] = `
<div
  className="page-header"
>
  <div
    className="content-container"
  >
    <h1
      className="page-header__title"
    >
      Viewing 
      <span>
        expense
      </span>
       expenses totalling 
      <span>
        $1.23
      </span>
    </h1>
    <div
      className="page-header__actions"
    >
      <Link
        className="button"
        to="/create"
      >
        Add Expense
      </Link>
    </div>
  </div>
</div>
`;

exports[`should correctly render Expenses summary with multiple expense  1`] = `
<div
  className="page-header"
>
  <div
    className="content-container"
  >
    <h1
      className="page-header__title"
    >
      Viewing 
      <span>
        expenses
      </span>
       expenses totalling 
      <span>
        $12,398.99
      </span>
    </h1>
    <div
      className="page-header__actions"
    >
      <Link
        className="button"
        to="/create"
      >
        Add Expense
      </Link>
    </div>
  </div>
</div>
`;
